BDArmory/UI/BDATeamIcons.cs

```diff
@@ -492,6 +495,15 @@ void UpdateStyles(bool forceUpdate = false)
                 mIStyle.normal.textColor = temp;
                 Missilecolor.a = Opacity;
             }
+            if (forceUpdate || BDTISettings.ICONSCALE != oldIconScale)
+            {
+                textScale = Mathf.Max(10, Mathf.CeilToInt(10 * BDTISettings.ICONSCALE)); //Would BD_UI_SCALE make more sense here?
```

Comment:
Maybe, but I think you'd still also want to have the factor of `BDTISettings.ICONSCALE` independent of `BDArmorySettings.UI_SCALE` so that you can set the icons as large as you want without having to scale the entire UI.
You'd want to trigger a `UpdateStyles(true)` call when `BDArmorySettings.UI_SCALE` changes if that factor is added though.